mm/alignment.py

```diff
@@ -551,9 +659,10 @@ def run(self) -> None:
                 self.hyp_silence_phone,
                 self.hyp_upsample_factor,
             )
-            assert len(hyp_word_starts) == len(
-                hyp_word_ends
-            ), f"Found different number of word starts ({len(hyp_word_starts)}) than word ends ({len(hyp_word_ends)}). Something seems to be broken."
+            assert len(hyp_word_starts) == len(hyp_word_ends), (
```

Comment:
Please take care that your ide does not reformat lines that do not belong to the PR.

e.g. set "format only changed lines" or select black from environment as your formatting provider as opposed to black that ships with the extension or ruff.

Reply:
Thanks for the suggestion.